### ISSUE TITLE
Mixed space and tab in tools/randsearch.py

### ISSUE THREAD

**Issue by DomML** [sjdv1982/attract#5]
![image](https://user-images.githubusercontent.com/25391513/75989339-62732a80-5ef3-11ea-8c69-0eb487757fd4.png)

**Comment by sjdv1982** [comment]
Yes, that code has not been touched since mid 2015. Python2 does not care about mixed tabs and spaces, but Python3 does. For now, ATTRACT uses python2; porting it to python3 is on the TODO list

**Comment by DomML** [comment]
Ok, thank you.
